.github/run_test.sh

```diff
@@ -8,48 +8,49 @@ C7_CONTAINER_IMAGE="quay.io/linux-system-roles/c7-network-role"
 C9S_CONTAINER_IMAGE="quay.io/linux-system-roles/c9s-network-role"
 PODMAN_OPTS="--systemd=true --privileged"
 
-# FIXME: test_wireless has been removed because is not supported on CI
-# container EL7. We need to add this back for EL8 or greater.
-EXCLUDE_TESTS='
+# exclude bond tests since missing the bonding kernel module
+# exclude tests/tests_wireless_nm.yml since failing to load mac80211_hwsim kernel
+# module to mock a wifi network
+EXCLUDE_TESTS_C7='
+-e tests/tests_auto_gateway_initscripts.yml
+-e tests/tests_bond_deprecated_initscripts.yml
+-e tests/tests_bond_initscripts.yml
+-e tests/tests_bond_removal_initscripts.yml
+-e tests/tests_team_nm.yml
+-e tests/tests_unit.yml
+-e tests/tests_wireless_nm.yml
+'
+
+# exclude bond tests since missing the bonding kernel module
+# exclude tests/tests_wireless_wpa3_owe_nm.yml tests/tests_wireless_wpa3_sae_nm.yml
+# since failing to install hostapd and mac80211_hwsim kernel module
+EXCLUDE_TESTS_C8S='
 -e tests/tests_802_1x_updated_nm.yml
 -e tests/tests_auto_gateway_initscripts.yml
 -e tests/tests_bond_deprecated_initscripts.yml
 -e tests/tests_bond_deprecated_nm.yml
 -e tests/tests_bond_initscripts.yml
 -e tests/tests_bond_removal_initscripts.yml
--e tests/tests_default_initscripts.yml
--e tests/tests_default_nm.yml
--e tests/tests_default.yml
--e tests/tests_ethernet_initscripts.yml
--e tests/tests_ethernet_nm.yml
--e tests/tests_eth_pci_address_match_nm.yml
--e tests/tests_ethtool_coalesce_initscripts.yml
--e tests/tests_ethtool_features_initscripts.yml
--e tests/tests_ethtool_ring_initscripts.yml
--e tests/tests_helpers_and_asserts.yml
 -e tests/tests_integration_pytest.yml
--e tests/tests_ipv6_dns_search_nm.yml
--e tests/tests_ipv6_initscripts.yml
+-e tests/tests_team_nm.yml
+-e tests/tests_unit.yml
+-e tests/tests_wireless_wpa3_owe_nm.yml
+-e tests/tests_wireless_wpa3_sae_nm.yml
+'
+
+# exclude tests_provider_nm.yml and tests_regression_nm.yml since no package
+# network-scripts available
+# exclude tests_wireless_wpa3_owe_nm.yml and tests_wireless_wpa3_sae_nm.yml since
+# destination directory /etc/hostapd not found in the container
```

Comment:
hostapd package is not installed for C9s.

Reply:
But even if we install the hostapd in C9s, the tests still fail in installiing mac80211_hwsim kernel module from here https://koji.mbox.centos.org/pkgs/packages/kernel/, because the container shares the kernel with ubuntu host.

Reply:
The following extracted from `tests/tasks/setup_mock_wifi_wpa3_owe.yml`, maybe we should not change into ansible_distribution_major_version != '7', because we can not install the mac80211_hwsim kernel module in C9s

```yaml
# Even though hostapd can be installed via EPEL 8, Opportunistic Wireless Encryption
# (OWE) has not been enabled by default. To warrant the test support on RHEL(dist-tag),
# we setup hostapd copr repo to enable OWE option.
- name: install hostapd and mac80211_hwsim kernel module in CentOS 8
  shell: |
    dnf -y copr enable liangwen12year/hostapd-owe
    dnf -y install hostapd
    release=$(uname -r)
    tmp="${release/-//}"
    tmp="${tmp/.x//x}"
    dnf -y install https://koji.mbox.centos.org/pkgs/packages/kernel/$tmp/kernel-core-$release.rpm
    dnf -y install https://koji.mbox.centos.org/pkgs/packages/kernel/$tmp/kernel-modules-$release.rpm
    dnf -y install https://koji.mbox.centos.org/pkgs/packages/kernel/$tmp/kernel-modules-internal-$release.rpm
  when:
    - ansible_distribution_major_version == '8'
    - ansible_distribution == 'CentOS'
```